src/contract.rs

```diff
@@ -127,7 +122,7 @@ impl StandardReferenceTrait for StandardReference {
         read_admin(&env).require_auth();
 
         add_relayers(&env, &addresses);
-        bump_instance_ttl_to_max(&env);
+        bump_instance_ttl(&env);
```

Comment:
If we need to `bump_instance_ttl` on every add relayers can we move this to add_relayers?